### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by NajaWulff** [Notalib/LYT#424]
Simon, Michael Drud and I have done som general testing on release #10 with JAWS 13 and 10. We have found the following flaws and pointers for futher adjustments of the player: 

General: 
- The 'Mine bøger' button - alt text should be called 'Tilbage til boghylde' `FIXED`
- The 'Søg' page needs another title, since there is also a button with the same name on that page `FIXED`
- The page that is presented to the user after she has clicked on a book should have the same title 
  as that book - currently it is simply called 'bog', which is uninformative.  This could possibly also be achieved simply by moving automatic focus on that page to the book's title. `FIXED`
- In the player, the 'Indhold og bogmærker' button should be called 'Infholdsfortegnelse' `FIXED`
- The 'Fjern bog' link - alt text should be called 'Fjern %title' `FIXED`
- Bookmarks should be prefixed with a number instead of having their placement in the audio file as their title `FIXED`  See #292 + #293
- The automatic focus on the login page should be on the 'Brugernavn' edit field `FIXED`
- The automatic focus in the player should be the 'Afspil' button `FIXED`
- The automatic focus on the search page should be the search field `FIXED`
- The automatic focus on the bookshelf should be on the top book `FIXED`
